tests/customer/logout/customerCanLogout.spec.js

```
@@ -19,7 +19,8 @@ test('Assert correct customer Logout', async ({ page }) => {
   const customerLoginPage = new CustomerLoginPage(page);
   const accountPage = new CustomerAccountPage(page);
 
-  await bankHomePage.open();
+  await bankHomePage.openLoginPage();
+  await bankHomePage.waitLoginPage()
```

waitLoginPage() waits for a relative URL. To make the check explicit and robust, ensure your config has baseURL set or use an assertion with the full expected URL (or `expect(page).toHaveURL('https://www.globalsqa.com/angularJs-protractor/BankingProject/#/login')`). This avoids silent failures when baseURL is not configured.